### PR TITLE
Added Helper script for Application gateway end to end encryption 

### PR DESCRIPTION
**Pull request by v-rahkuma**
Added Helper script for Application gateway end to end encryption

### CODE REVIEW

File: Control coverage/Feature/ApplicationGateway.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 - [Azure_ApplicationGateway_NetSec_Enable_WAF_Configuration](#azure_applicationgateway_netsec_enable_waf_configuration)
 - [Azure_ApplicationGateway_NetSec_Enable_DDoS_Protection](#azure_applicationGateway_netsec_enable_ddos_protection)
-
+- [Azure_ApplicationGateway_DP_Enable_EndToEnd_Encryption_Trial](#Azure_ApplicationGateway_DP_Enable_EndToEnd_Encryption_Trial)
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/ApplicationGateway.md:9]: I thought we didn't put documentation for trial controls? Is this correct?

File: Control coverage/Feature/ApplicationGateway.md [hunk 2]
```diff
@@ -130,5 +130,109 @@ Enabling DDOS on Vnet of Application Gateway, provides protection and defense fo
...
+> SSL policy if being used, is not set to TLS 1.2 or higher.
+> 
+> **Error:** 
+> There was an error fetching Configuration details of Application Gateway.
+> 
+### Recommendation 
+
+- **Azure Portal** 
+
+	 Use the Azure portal to configure end to end encryption on the Application Gateway.
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/ApplicationGateway.md:163]: If we're giving a recomendation then we might want to be specific.

Comment (aakashpahwa-msft) [reply Control coverage/Feature/ApplicationGateway.md:163]: And if we can't then might as well remove it.

**Review by aakashpahwa-msft — CHANGES_REQUESTED** [review]

File: Control coverage/Feature/ApplicationGateway.md (hunk 1 above)
Comment (v-rahkuma) [reply Control coverage/Feature/ApplicationGateway.md]: Removed trial

File: Control coverage/Feature/ApplicationGateway.md (hunk 2 above)
Comment (v-rahkuma) [reply Control coverage/Feature/ApplicationGateway.md]: Added detailed recommendation

**Review by Aboli-msft — APPROVED** [review]

**Review by aakashpahwa-msft — APPROVED** [review]

**Pull request merged**